api/src/services/dwc-service.ts

```diff
@@ -83,11 +83,16 @@ export class DarwinCoreService extends DBService {
 
       await this.create_step6_transformAndUploadMetaData(submissionId, dataPackageId);
 
-      await this.create_step7_normalizeSubmissionDWCA(submissionId);
+      const isOnlyMetadata = await this.isSubmissionMetadataOnly(submissionId);
 
-      await this.create_step8_runSpatialTransforms(submissionId);
+      // skip these if the submission only contains metadata
```

Comment:
nit: can you elaborate on this comment. Include something about how these steps all involve processing the darwin core observation data, and so can be skipped if the incoming dataset doesn't include it.

Reply:
added a bit more to clarify